__tests__/fixtures/new-architecture/basic-functional-component-with-ref/code.js

```diff
@@ -0,0 +1,6 @@
+import { View } from 'react-native';
+import { useRef } from 'react';
+function Component() {
+  const ref = useRef(null);
```

Comment:
Can you add a test that [includes a functional ref](https://react.dev/reference/react-dom/components/common#ref-callback)?

Reply:
added